addons/website_blog/static/src/js/components/translator.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module **/
+
+import { WebsiteTranslator } from '@website/components/translator/translator';
+import { patch } from 'web.utils';
+
+patch(WebsiteTranslator.prototype, 'website_blog_translator', {
+    /**
+     * @override
+     */
+    _beforeEditorActive() {
+        this._super(...arguments);
+        $(this.websiteService.pageDocument).find('[data-translate-error-tooltip]').tooltip({
+            container: this.websiteService.pageDocument.body,
+            trigger: 'click',
+            delay: {'show': 0, 'hide': 0},
+            title: function () {
+                return $(this).data('translate-error-tooltip');
+            },
+        });
+    }
```

Comment:
I tried pretty much everything, including all possible `trigger` properties, using `popover` instead of `tooltip`, binding the tooltip directly on the element or using the `selector` property and calling `.tooltip()` on the body etc.

I can't make it so it closes when you click anywhere on screen. I can only make it work with the default behavior: it closes only when you click on the text which opened it.

Probably something around this line messing around: https://github.com/odoo/odoo/blame/120046ad7d915fd71b4b5f55ad2c5269406bd251/addons/website/static/src/client_actions/website_preview/website_preview.js#L420-L422

It captures the click before the tooltip clicks kicks in.

IMO not worth spending more time, it's fine that way?